provider/kubernetes/client.go

```diff
@@ -165,6 +169,13 @@ func (c *clientImpl) GetIngresses() []*extensionsv1beta1.Ingress {
 			result = append(result, ing)
 		}
 	}
+
+	sort.Slice(result, func(i, j int) bool {
+		i1 := result[i]
+		i2 := result[j]
```

Comment:
```suggestion
```

Reply:
True but the maps are fine it is only the TLS which is a slice (which has an order) that is offending.